functions/mqhandlers.go

```diff
@@ -307,7 +307,7 @@ func handleEndpointDetection(peerUpdate *models.HostPeerUpdate) {
 					peerPubKey,
 					peerInfo.ProxyListenPort,
 				); err != nil { // happens v often
-					slog.Error("failed to check for endpoint on peer", "peer", peerPubKey, "error", err)
+					slog.Warn("failed to check for endpoint on peer", "peer", peerPubKey, "error", err)
```

Comment:
I think that this could be at Debug level.

Reply:
agreed

Reply:
https://github.com/gravitl/netclient/pull/478/commits/696830ab645c08fbd457590734de0a5268930beb